src/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface Translator {
+  get: (str: string, args?: string[]) => string;
```

Comment:
this should be a type in nebula?

Reply:
unfortunately, no. nebula only exports galaxy, not Translator specifically.

Reply:
I see this is the way pivot does it as well

Reply:
good